ngraph/core/src/op/reduce_sum.cpp

```diff
@@ -73,5 +75,15 @@ bool op::v1::ReduceSum::evaluate(const HostTensorVector& outputs,
                                  const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v1_ReduceSum_evaluate);
-    return reduce_sum::evaluate_sum(inputs[0], outputs[0], get_reduction_axes(), get_keep_dims());
+    auto reduction_axes = get_reduction_axes();
+
+    if (!ngraph::op::is_constant(input(1).get_node()) && inputs.size() > 1)
```

Comment:
I think that we need to check that there are more than 1 input before using it. So, please, swap the arguments of `&&`